freqtrade/freqtradebot.py

```diff
@@ -1145,14 +1145,15 @@ def get_real_amount(self, trade: Trade, order: Dict, order_amount: float = None)
             return order_amount
         amount = 0
         fee_abs = 0
+        trade_base_currency = self.exchange.get_pair_base_currency(trade.pair)
```

Comment:
already assigned above?

Reply:
Just testing how good the review is :laughing: :+1: 

great catch :+1:

Reply:
🤣